register/models.py

```diff
@@ -30,6 +30,7 @@ class User(models.Model):
     )
     username = models.CharField(max_length=20, null=True, blank=False)
     password = models.CharField(max_length=256)
+    is_active = models.BooleanField(default=False)
```

Comment:
this change should also be reflected in the test cases otherwise they will fail.

Reply:
the coverage test was 92% so does that satisfy this criteria?

Reply:
since there are unit tests written for models, changing a model might make the tests fail. Just wanted to confirm that the tests are passing.